lib/views/home.dart

```diff
@@ -0,0 +1,296 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_second_task_spotify_clone/navigations/tabbar.dart';
+import 'package:flutter_second_task_spotify_clone/widgets/album_card.dart';
+import 'package:flutter_second_task_spotify_clone/widgets/song_card.dart';
+
+class HomeView extends StatefulWidget {
+  const HomeView({Key? key}) : super(key: key);
+
+  @override
+  _HomeViewState createState() => _HomeViewState();
+}
+
+class _HomeViewState extends State<HomeView> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Stack(
+        alignment: Alignment.topLeft,
+        children: [
+          Container(
+            width: MediaQuery.of(context).size.width,
+            height: MediaQuery.of(context).size.height * .6,
+            decoration: const BoxDecoration(
+              color: Color(0xFf1C7A74),
+            ),
+          ),
+          SingleChildScrollView(
+            physics: const BouncingScrollPhysics(),
+            child: Container(
+              decoration: BoxDecoration(
+                gradient: LinearGradient(
+                  begin: Alignment.topCenter,
+                  end: Alignment.bottomCenter,
+                  colors: [
+                    Colors.black.withOpacity(0),
+                    Colors.black.withOpacity(.9),
+                    Colors.black.withOpacity(1),
+                    Colors.black.withOpacity(1),
+                    Colors.black.withOpacity(1),
+                  ],
+                ),
+              ),
+              child: SafeArea(
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  mainAxisAlignment: MainAxisAlignment.start,
+                  children: [
+                    const SizedBox(height: 40),
+                    Padding(
+                      padding: const EdgeInsets.symmetric(horizontal: 16),
+                      child: Row(
+                        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                        children: [
+                          Text(
+                            "Recently Played",
+                            style: Theme.of(context).textTheme.headline6,
+                          ),
+                          Row(
+                            children: const [
+                              Icon(Icons.history),
+                              SizedBox(width: 16),
+                              Icon(Icons.settings),
+                            ],
+                          )
+                        ],
+                      ),
+                    ),
+                    SingleChildScrollView(
+                      scrollDirection: Axis.horizontal,
+                      physics: const BouncingScrollPhysics(),
+                      padding: const EdgeInsets.all(16),
+                      child: Row(
+                        children: const [
+                          AlbumCard(
```

Comment:
You can also try to put cards into array variable, and use ListView builder to generate this view (I also did not manage to fully apply this to my project anyway..)